ses

```diff
@@ -130,6 +130,25 @@ if [ -d /etc/salt ]; then
 fi
 
 #############################################################
+section_header "NFS-Ganesha config"
+
+if [ -e /etc/ganesha/ganesha.conf ]; then
+    mkdir -p $LOG/ceph/ganesha/
+    cp /etc/ganesha/ganesha.conf $LOG/ceph/ganesha/ganesha.conf
+    plugin_message "NFS-Ganesha config copied to ceph/ganesha/ subdirectory"
+fi
+
+#############################################################
+section_header "NFS-Ganesha log files"
+
+if [ -e /var/log/ganesha.log ]; then
+    mkdir -p $LOG/ceph/ganesha/log
+    cp /var/log/ganesha.log $LOG/ceph/ganesha/log
```

Comment:
Not really, as the only config we're saving right now is from `/etc/ceph/`, but that's a good idea, let's go with `$LOG/ceph/conf/ganesha/`